code/modules/power/cable.dm

```diff
@@ -39,13 +53,8 @@
 		if(!(linked_dirs & dir))
 			continue
 		new_dir_count++
-	if(new_dir_count == 1)
-		knotted = TRUE
-	else
-		knotted = FALSE
 	if(new_dir_count > 2)
 		WARNING("Cable has more than 2 directions on [loc.x],[loc.y],[loc.z]")
```

Comment:
This prints to world, it should probably crash or have some other sort of handling.